docs/BedWars1058/hooks/vault-hook.md

```diff
@@ -24,4 +24,14 @@ chat:
   global: '{vPrefix}&6[SHOUT] {team} &7{player}&f{vSuffix}: {message}'
   team: '{vPrefix}&f{level}{team}&7 {player}{vSuffix} {message}'
   spectator: '{vPrefix}&7[SPECTATOR] {player}{vSuffix}: {message}'
-```
\ No newline at end of file
+```
+
+## Economy
+**BedWars1058** is also providing Vault Economy plugins support since the first version ever released.
+
+You can use all the Economy plugins with [Vault](https://dev.bukkit.org/projects/vault) support like [Coins](https://www.spigotmc.org/resources/coins-1-8-x-1-12-x-mysql-sqlite-api.48536/) and any other economy plugins based on [Vault](https://dev.bukkit.org/projects/vault).
+
+## Placeholders
```

Comment:
You should make this a sub-heading (### instead of ##)

Also make sure to change the Placeholders heading for the chat part